students/JoeNunnelley/session09/donor_models.py

```diff
@@ -0,0 +1,121 @@
+"""
+The Various Mailroom Classes
+- Donor
+- Donation
+- DonorCollection
+"""
+
+import datetime
+
+class Donor:
+    """The Donor Class that contains donation information"""
+    def __init__(self, name, _donations=None):
+        self.__donations = _donations
```

Comment:
Since you are using the Donation class to track Donations, you may want to do some sort of error checking here otherwise you could add varying types